tests/testthat/dummy_packages/cp1252/R/cp1252.R

```diff
@@ -0,0 +1,4 @@
+# This file is encoded in Cp-1252
+
+# Comment containing non-ASCII �
```

Comment:
non-ASCII in test files is OK right?

Reply:
OK for? devtools::check()` didn't complain about it IINM.